docs/management_api.yml

```diff
@@ -519,9 +516,7 @@ paths:
         200:
           description: OK
           schema:
-            type: array
-            items:
-              $ref: "#/definitions/DeviceWithImage"
+            example: []
```

Comment:
This is tricky and I'm happy you noticed it 🙃

If I understand this correctly, then when we build api docs from swagger it generates code samples using only `path` parameters (in this case it's `/api/management/v1/deployments/deployments/{deployment_id}/devices/list`) and after making this api call empty list is expected returned output.

Showing in example device definition is definitely valuable, but it requires `status` query to be used, which is not required parameter. One of the possible options is to change default behaviour and send all devices list from the deployment when `status` query is empty. What do you think about that?

Reply:
>  it requires status query to be used

it does not require status query parameter;

it's possible to get empty list here, just because devices didn't ask for an update yet, and this endpoint will show devices only after devices will ask for an update and become part of this deployment

example query:
https://staging.hosted.mender.io/api/management/v1/deployments/deployments/df17b139-3bb4-4f2d-8985-9f934842a479/devices/list
and result:
`[{"created":"2023-02-24T11:19:32.955Z","finished":"2023-02-24T11:21:05.224Z","status":"success","id":"4c0370ba-a92b-4a29-ada0-ff7493b92dc1","log":false,"attempts":1,"delta_job_id":""},{"created":"2023-02-24T11:19:32.955Z","finished":"2023-02-24T11:21:03.247Z","status":"success","id":"6aafbac4-2179-40bd-b5da-0bafa6fa32ef","log":false,"attempts":1,"delta_job_id":""},{"created":"2023-02-24T11:19:32.955Z","finished":"2023-02-24T11:21:11.783Z","status":"success","id":"bcd7053e-4f00-4d9d-b11c-167074cf0620","log":false,"attempts":1,"delta_job_id":""},{"created":"2023-02-24T11:19:32.955Z","finished":"2023-02-24T11:21:07.16Z","status":"success","id":"d87e1d29-5ae8-4622-b19c-55427a21497a","log":false,"attempts":1,"delta_job_id":""},{"created":"2023-02-24T11:19:32.955Z","finished":"2023-02-24T11:21:09.159Z","status":"success","id":"dadabc1e-1812-4701-953e-d3ef6aec4b51","log":false,"attempts":1,"delta_job_id":""}]`